src/components/Detail/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams, useNavigate } from 'react-router-dom';
+import '../../css/details.css';
+import { fetchDetail } from '../../redux/detailReducer';
+import { formToggle } from '../../redux/reservations/reservationReducer';
+
+const Detail = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { carid } = useParams();
```

Comment:
- [ ] **(OPTIONAL)** In React, it's a better practice to use camelCase for variable names, so this would be `carId`.